developer/services/qrexec3.md

```diff
@@ -128,47 +111,34 @@ These files contain lines with the following format:
 
     srcvm destvm (allow|deny|ask)[,user=user_to_run_as][,target=VM_to_redirect_to]
 
-You can specify srcvm and destvm by name, or by one of `$anyvm`, `$dispvm`,
-`dom0` reserved keywords (note string `dom0` does not match the `$anyvm`
-pattern; all other names do). Only `$anyvm` keyword makes sense in srcvm
-field (service calls from dom0 are currently always allowed, `$dispvm`
-means "new VM created for this particular request," so it is never a
-source of request). Currently there is no way to specify source VM by
-type. Whenever a rpc request for action X is received, the first line in
-`/etc/qubes-rpc/policy/X` that match srcvm/destvm is consulted to determine
-whether to allow rpc, what user account the program should run in target VM
-under, and what VM to redirect the execution to. Note that if the request is
-redirected (`target=` parameter), policy action remains the same - even if
-there is another rule which would otherwise deny such request. If the policy
-file does not exist, user is prompted to create one; if still there is no
-policy file after prompting, the action is denied.
-
-In the target VM, the `/etc/qubes-rpc/RPC_ACTION_NAME` must exist, containing
-the file name of the program that will be invoked, or being that program itself
-- in which case it must have executable permission set (`chmod +x`).
+You can specify srcvm and destvm by name or by one of three reserved keywords: `$anyvm`, `$dispvm`, and `dom0` (without the `$`).
+Only `$anyvm` keyword makes sense in srcvm field.
```

Comment:
Since R3.2 is not supported anymore, this can be merged with "Extra keywords available in Qubes 4.0 and later" section. This means additionally `@tag:...` and `@type:...` keywords are valid here.